src/mds/populate.py

```diff
@@ -20,15 +19,11 @@ def parse_args(argv: List[str]) -> Namespace:
 
     parser = argparse.ArgumentParser()
     parser.add_argument("--config", help="config file to use", type=str, required=True)
-    parser.add_argument(
-        "--hostname", help="hostname of server", type=str, default="localhost"
-    )
-    parser.add_argument("--port", help="port of server", type=int, default=6379)
     known_args, unknown_args = parser.parse_known_args(argv)
     return known_args
 
 
-async def populate_metadata(name: str, common, results):
+async def populate_metadata(name: str, common, results, populate_to_temp_index=False):
```

Comment:
There isn't a requirement for this to be backward compatible per se, but I thought it would be nice to preserve the ability to directly update the es index instead of going with the create temporary index/cloning.  Future functions may want this ability.